index.js

```diff
@@ -14,11 +14,13 @@ let mailer = require('mailgun-js');
 
 // Constructors
 
-function Mailer (config) {
-  this.config = config;
+function Mailer (domain, key, live = false) {
+  this.domain = domain;
```

Comment:
do these need to be saved in `this` if they're not being used again?

Reply:
i cant access live variable without this as we are using it in a send function

Reply:
ok, but `domain` and `key` don't have to be saved, right?

Reply:
yes right